install.sh

```diff
@@ -9,16 +56,22 @@ elif [[ ${OS} == 'Darwin' ]]; then
    brew install npm
 else
     echo 'Windows currently not supported for development'
+    exit;
+fi
+
+if [ -z "${FRONTEND}" ] && [ -z "${BACKEND}" ]; then
+   install_backend
+   install_frontend
 fi
 
-# Creating python virtual env and activates it
-python3 -m venv venv
-source venv/bin/activate
 
-# Installs all required backend python modules to run server.
-pip3 install --upgrade -r ./server/requirements.txt
+if [ -n "${BACKEND}" ]; then
+   install_backend
+fi
+
+if [ -n "${FRONTEND}" ]; then
+   install_frontend
+fi
 
-# Installs npm node_modules
-pushd deerhunt
-npm install
-popd
+printf "\n\n\nInstallation finished. Don't forget to activate venv by doing:\n"
+printf "   $ source venv/bin/activate\n\n"
```

Comment:
Newline here, usually we let it slide but this is bash so i think it's useful to follow bash convention.